### PR TITLE
gremlin: add BothV step

### PR DESCRIPTION
**Pull request by safchain**
This allow to retrieves in and out
nodes of an edge.

ex: G.E().BothV()

### CODE REVIEW

File: topology/graph/traversal/traversal.go [hunk 1]
```diff
@@ -1567,6 +1567,44 @@ func (te *GraphTraversalE) OutV(s ...interface{}) *GraphTraversalV {
...
+	if te.error != nil {
+		return &GraphTraversalV{error: te.error}
+	}
+
+	metadata, err := SliceToMetadata(s...)
+	if err != nil {
+		return &GraphTraversalV{error: err}
+	}
+
+	ntv := &GraphTraversalV{GraphTraversal: te.GraphTraversal, nodes: []*graph.Node{}}
```
Comment (lebauce) [inline topology/graph/traversal/traversal.go]: nit: ntv := NewGraphTraversalV(te.GraphTraversal, []*graph.Node{})

File: topology/graph/traversal/traversal.go [hunk 2]
```diff
@@ -1567,6 +1567,44 @@ func (te *GraphTraversalE) OutV(s ...interface{}) *GraphTraversalV {
...
+	it := te.GraphTraversal.currentStepContext.PaginationRange.Iterator()
+
+	te.GraphTraversal.RLock()
+	defer te.GraphTraversal.RUnlock()
+
+	for _, e := range te.edges {
+		parents, children := te.GraphTraversal.Graph.GetEdgeNodes(e, graph.Metadata{}, metadata)
```
Comment (lebauce) [inline topology/graph/traversal/traversal.go]: s/graph.Metadata{}/nil

Comment (lebauce) [inline topology/graph/traversal/traversal.go]: GetEdgeNodes(e, metadata, metadata) ?

File: topology/graph/traversal/traversal.go (hunk 1 above)
Comment (safchain) [reply topology/graph/traversal/traversal.go]: done

File: topology/graph/traversal/traversal.go (hunk 2 above)
Comment (safchain) [reply topology/graph/traversal/traversal.go]: yep good point

**Comment by safchain** [comment]
rerun unit-tests

**Pull request merged**